### PR TITLE
Demo pages (GCOM-1149)

### PR DESCRIPTION
**Pull request by Jessevdpoel**

### CODE REVIEW

File: examples/magento-graphcms/pages/test/icons.tsx [hunk 1]
```diff
@@ -34,6 +35,7 @@ export default function IconsPage() {
 
   return (
     <Container>
+      <LayoutHeader />
```
Comment (paales) [inline examples/magento-graphcms/pages/test/icons.tsx:38]: The LayoutHeader doesn't belong in a container

File: examples/magento-graphcms/pages/test/minimal-page-shell/[[...url]].tsx [hunk 2]
```diff
@@ -82,6 +83,9 @@ export function LayoutDemo(props: LayoutDemoProps) {
 
   return (
     <>
+      <LayoutTitle>
+        <Trans id='Layout demo' />
```
Comment (paales) [inline examples/magento-graphcms/pages/test/minimal-page-shell/[[...url]].tsx:87]: Remove translations from test pages, this impacts bundle size

File: examples/magento-graphcms/pages/test/[[...url]].tsx [hunk 3]
```diff
@@ -1,18 +1,40 @@
...
+      <LayoutDemo baseUrl='/test' />
+      <Divider />
+      <LayoutTitle
+        sx={(theme) => ({
+          '&.gutterBottom': {
+            marginBottom: theme.spacings.sm,
+          },
+        })}
+      >
+        <Trans id='Links to components' />
```
Comment (paales) [inline examples/magento-graphcms/pages/test/[[...url]].tsx:28]: Remove translations from test pages, this impacts bundle size

File: examples/magento-graphcms/pages/test/[[...url]].tsx [hunk 4]
```diff
@@ -1,18 +1,40 @@
...
+      </LayoutTitle>
+      <Container maxWidth='md' sx={{ display: 'grid', gridColumns: '1' }}>
+        <Link href='/test/buttons'>Buttons</Link>
+        <Link href='/test/icons'>Icons</Link>
+        <Link href='/test/slider'>Slider</Link>
+        <Link href='/test/typography'>Typography</Link>
```
Comment (paales) [inline examples/magento-graphcms/pages/test/[[...url]].tsx:34]: Missing number-inputs here

File: docs/framework/component-demos.md [hunk 5]
```diff
@@ -0,0 +1,8 @@
+---
+menu: Component demo's
+---
+
+# Demo's
+
+You can find a demo for the layout and different components on
+graphcommerce.vercel.app/test
```
Comment (paales) [inline docs/framework/component-demos.md:8]: Can we remove this page until we actually have something pretty/complete to show?

**Review by paales — CHANGES_REQUESTED** [review]

File: docs/framework/component-demos.md (hunk 5 above)
Comment (machielkreiken) [reply docs/framework/component-demos.md:8]: Just discussed with Paul. For now we will not provide a link in the documenation, untill we have a clearer purpose for these pages.

**Pull request merged**